pkg/telemetry/telemetry.go

```diff
@@ -31,6 +32,7 @@ func SendTelemetry(analysis *analyser.Analysis) {
 		Arch:           runtime.GOARCH,
 		TotalResources: analysis.Summary().TotalResources,
 		TotalManaged:   analysis.Summary().TotalManaged,
+		Duration:       uint(analysis.Duration.Seconds() + 0.5),
```

Comment:
Why not using the float64 type here ?

Reply:
On the backend we expect to retrieve an `uint`

Reply:
Why adding 0.5 here ?

Reply:
This is an elegant way to round to the nearest integer. `1.4` will become `1.9` and casted to `uint 1` and `1.5` will become `2.0` casted to `uint 2`